common/tailwind/tailwind.config.js

```diff
@@ -310,7 +310,6 @@ const config = {
         '../../ee/frontend/**/*.{ts,tsx}',
         '../../frontend/src/index.html',
         '../../products/**/frontend/**/*.{ts,tsx}',
-        '../../common/**/src/**/*.{ts,tsx}',
         '../../common/**/frontend/**/*.{ts,tsx}',
```

Comment:
This folder doesn't match anything, there's no `frontend` folder inside `common`